src/Tools/dotnet-monitor/Extensibility/ToolsExtensionRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Extensibility
+{
+    internal class ToolsExtensionRepository : ExtensionRepository
+    {
+        private readonly string _targetFolder;
+        private readonly IFileProvider _fileSystem;
+        private readonly ILoggerFactory _loggerFactory;
```

Comment:
This isn't used other than for creating a single `ILogger` instance. Can we change it so that the ctor takes `ILogger<ProgramExtension>` and have the service creation get that instead of getting `ILoggerFactory`?